spec/presenters/status_relationships_presenter_spec.rb

```diff
@@ -107,18 +121,30 @@
 
       it 'sets @filters_map to filter top-level status' do
         matched_filters = presenter.filters_map[statuses[0].id]
-        expect(matched_filters.size).to eq 1
 
-        expect(matched_filters[0].filter.title).to eq 'filter1'
-        expect(matched_filters[0].status_matches).to eq [statuses[0].id]
+        expect(matched_filters)
+          .to be_an(Array)
+          .and have_attributes(size: 1)
+          .and contain_exactly(
+            have_attributes(
+              filter: have_attributes(title: 'filter1'),
+              status_matches: include(statuses.first.id)
```

Comment:
Maybe it should be more specific and match `[statuses.first.id]` exactly instead of just `include`?